var/spack/repos/builtin/packages/openfst/package.py

```diff
@@ -34,17 +35,16 @@ class Openfst(AutotoolsPackage):
     conflicts('%gcc@6:', when='@:1.6.1')
 
     variant('far', default=False, description="Enable FAR support")
+    variant('python', default=False, description="Enable python extension")
+
+    depends_on('python@3.6:', when='+python')
+
+    extends('python', when='+python')
 
     # Patch openfst-1.4.1 for kaldi@c024e8
     # See https://github.com/kaldi-asr/kaldi/blob/c024e8aa0a727bf76c91a318f76a1f8b0b59249e/tools/Makefile#L82-L88
     patch('openfst-1.4.1.patch', when='@1.4.1-patch')
     patch('openfst_gcc41up.patch', when='@1.4.1-patch')
 
     def configure_args(self):
-        args = []
-        spec = self.spec
-        if '+far' in spec:
-            args.append('--enable-far')
-        else:
-            args.append('--disable-far')
-        return args
+         return self.enable_or_disable('far') + self.enable_or_disable('python')
```

Comment:
I think there's a space too much:
```suggestion
        return self.enable_or_disable('far') + self.enable_or_disable('python')
```

Reply:
Addition space removed.